public/css/icinga/forms.less

```diff
@@ -517,3 +517,26 @@ form.icinga-form .form-info {
 .flyover-content .control-label-group {
   text-align: left;
 }
+
+.control-group > label {
+  text-align: center;
+}
```

Comment:
This is

a) too broad. All form styles are wrapped in either `.icinga-form` (layout) or `.icinga-controls` (style).
b) There are other cases where a label is directly in a `.control-group` and this doesn't make any sense

Remove this and wrap the theme mode's labels with `span`s and style those instead.

Reply:
✅